Automattic/jetpack
# Protect: Update footer designs

#### Description
This PR implements the necessary design changes to the Protect admin footer, as part of the Protect and Scan integration project.

#### Changes proposed in this Pull Request:

* Updates conditional footer content and designs
* Adds analytics tracking for the click event

#### TODO items:
* Update required plan, when available
* Add appropriate links where necessary, when available

#### Other information:

#### Jetpack product discussion


- 1201069996155224-as-1203034583317312
- p1HpG7-gSG-p2

#### Does this pull request change what data or activity we track or use?


- No

#### Testing instructions:






* Checkout this branch
* Run your Jurassic Tube
* Activate Protect
* Proceed with a Free connection and allow the initial report to generate
   * View the footer of the corresponding status page and compare against Figma designs
   * Enable mobile view and compare
   * Ensure buttons and links respond as expected where applicable
* Proceed with a Security Bundle upgrade or modify the `( hasRequiredPlan )` check in the code to reverse the logic
   * View the footer again and compare against Figma designs
   * Enable mobile view and compare
   * Ensure buttons and links respond as expected where applicable, noting that some links have been left with only a placeholder for the time being
